rt/rs/description-swagger-ui/src/main/java/org/apache/cxf/jaxrs/swagger/OsgiSwaggerUiResolver.java

```diff
@@ -47,6 +47,9 @@ public String findSwaggerUiRootInternal(String swaggerUiMavenGroupAndArtifact,
             if (bundle == null) {
                 return null;
             }
+            if (bundle.getState() != Bundle.ACTIVE) {
+				bundle.start();
```

Comment:
is it me or the indentation is mixed, space and tabs?

Reply:
Uh.... sorry, fixed. Was lazy and didn't edit it in the IDE but directly in GitHub and thought GitHub would be smart and tell me if indent uses the wrong characters. It doesn't.